validator/accounts/v2/new.go

```diff
@@ -0,0 +1,237 @@
+package v2
+
+import (
+	"context"
+	"errors"
+	"os"
+	"unicode"
+
+	"github.com/manifoldco/promptui"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+
+	"github.com/prysmaticlabs/prysm/validator/flags"
+	"github.com/prysmaticlabs/prysm/validator/keymanager/v2/direct"
+)
+
+var log = logrus.WithField("prefix", "accounts-v2")
+
+// WalletType defines an enum for either direct, derived, or remote-signing
+// wallets as specified by a user during account creation.
+type WalletType int
+
+const (
+	// DirectWallet defines an on-disk, encrypted keystore-capable wallet.
+	DirectWallet WalletType = iota
+	// DerivedWallet defines a hierarchical-deterministic wallet.
+	DerivedWallet
+	// RemoteWallet capable of remote-signing data.
+	RemoteWallet
+)
+
+const minPasswordLength = 8
+
+var walletTypeSelections = map[WalletType]string{
+	DirectWallet:  "Direct, On-Disk Accounts (Recommended)",
+	DerivedWallet: "Derived Accounts (Advanced)",
+	RemoteWallet:  "Remote Accounts (Advanced)",
+}
+
+// New creates a new validator account from user input. If a user
+// does not have an initialized wallet at the specified wallet path, this
+// method will create a new wallet and ask user for input for their new wallet's
+// available options.
+func New(cliCtx *cli.Context) error {
```

Comment:
Can this be renamed to `NewWallet`?

Reply:
This is meant to be used as `accounts.New`. The purpose of this function is to create a new validator account. Maybe we can rename to accounts.NewAccount instead?

Reply:
Yeah, that sounds good.